module/Photo/config/module.config.php

```diff
@@ -42,6 +42,19 @@
                             ),
                         ),
                     ),
+                    'year' => array(
```

Comment:
Wellicht duidelijk aangeven dat het hier om het schooljaar gaat, en niet om het jaar. Dit zorgde er voor dat ik andere dingen niet helemaal begreep in de commit